core-it-suite/src/test/java/org/apache/maven/it/ItUtils.java

```diff
@@ -19,7 +19,6 @@
  * under the License.
  */
 
-import org.junit.jupiter.api.Test;
```

Comment:
This is unrelated, but it's an unused import, so hopefully fine to remove it as part of this.